hbase-client/src/main/java/org/apache/hadoop/hbase/client/HTableMultiplexer.java

```diff
@@ -434,7 +434,8 @@ public FlushWorker(Configuration conf, ClusterConnection conn, HRegionLocation a
         HConstants.DEFAULT_HBASE_CLIENT_OPERATION_TIMEOUT);
       // Specify 0 retries in AsyncProcess because we need to reassign puts to different queues
       // if regions are moved.
-      this.ap = new AsyncProcess(conn, conf, rpcCallerFactory, rpcControllerFactory, 0);
+      this.ap = new AsyncProcess(conn, conf, rpcCallerFactory, rpcControllerFactory, 0,
+        Collections.emptyMap());
```

Comment:
do we need request attribute support here? Seems like maybe no since it is deprecated

Reply:
yea, let's leave it off HTableMultiplexer, since it's deprecated (and rather unmaintained afaict)